internal/common/conversion/schema_generation_test.go

```diff
@@ -265,6 +240,191 @@ func TestDataSourceSchemaFromResource(t *testing.T) {
 	assert.Equal(t, expected, ds)
 }
 
+func TestDataSourceSchemaFromResource_blocksToAttrs(t *testing.T) {
```

Comment:
Nice. Looks like we are covering most cases here.
[nit] A bit long names 😅 `block block` could be replaced with bb?

Reply:
it's true that they could be shorter, but these long names really helped while developing the functionality. 

changed in: https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2904/commits/99a6b0db0aabb6cc971d9271faac5d99a5745586